src/status_im/utils/types.cljs

```diff
@@ -13,4 +13,6 @@
 
 (defn json->clj [json]
   (when-not (= json "undefined")
-    (js->clj (.parse js/JSON json) :keywordize-keys true)))
+    (try
+      (js->clj (.parse js/JSON json) :keywordize-keys true)
+      (catch js/Error _ (when (string? json) json)))))
```

Comment:
This will not help actually, even if `json` is string but has wrong format exception will be thrown.